modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'json_schema/json_api_missing_attribute'
+require 'appeals_api/form_schemas'
+
+class AppealsApi::V2::DecisionReviews::HigherLevelReviewsController < AppealsApi::ApplicationController
+  include AppealsApi::JsonFormatValidation
+  include AppealsApi::StatusSimulation
+
+  skip_before_action :authenticate
+  before_action :validate_json_format, if: -> { request.post? }
+  before_action :validate_json_schema, only: %i[create validate]
+  before_action :new_higher_level_review, only: %i[create validate]
+  before_action :find_higher_level_review, only: %i[show]
+
+  FORM_NUMBER = '200996'
+  MODEL_ERROR_STATUS = 422
+  HEADERS = JSON.parse(
+    File.read(
+      AppealsApi::Engine.root.join('config/schemas/v1/200996_headers.json')
+    )
+  )['definitions']['hlrCreateParameters']['properties'].keys
+
+  def create
+    @higher_level_review.save
+    AppealsApi::HigherLevelReviewPdfSubmitJob.perform_async(@higher_level_review.id, 'V1')
+    render_higher_level_review
+  end
+
+  def validate
+    render json: validation_success
+  end
+
+  def schema
+    render json: AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(
+      AppealsApi::FormSchemas.new.schema(self.class::FORM_NUMBER)
+    )
+  end
+
+  def show
+    @higher_level_review = with_status_simulation(@higher_level_review) if status_requested_and_allowed?
+    render_higher_level_review
+  end
+
+  private
+
+  def validate_json_schema
+    validate_json_schema_for_headers
+    validate_json_schema_for_body
+  rescue JsonSchema::JsonApiMissingAttribute => e
+    render json: e.to_json_api, status: e.code
+  end
+
+  def validate_json_schema_for_headers
+    AppealsApi::FormSchemas.new.validate!("#{self.class::FORM_NUMBER}_HEADERS", headers)
```

Comment:
Just commenting here that we'll eventually want to pass `Common::Exceptions::DetailedSchemaErrors` into this initializer to use the better errors. We have a dedicated ticket for it but this is where that'll happen.